lib/filewatch/tail.rb

```diff
@@ -162,6 +162,8 @@ def _open_file(path, event)
           @files[path].sysseek(stat.size, IO::SEEK_SET)
           @sincedb[sincedb_record_uid] = stat.size
         end
+      elsif event == :create && @files[path]
+        @sincedb[sincedb_record_uid] = 0
```

Comment:
Not really because then if you set  start_position => end and move a big file into the glob it will not be read from the start